components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -138,13 +138,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
     }, [logsEmitter, props.workspaceId, workspaceInstance?.id, workspaceInstance?.status.phase]);
 
     return (
-        <div className="rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col mb-8">
+        <div className="relative rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col mb-8">
             <div className="h-96 flex">
                 <Suspense fallback={<div />}>
                     <WorkspaceLogs classes="h-full w-full" logsEmitter={logsEmitter} errorMessage={error?.message} />
                 </Suspense>
             </div>
-            <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
+            <div className="absolute bottom-0 h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
```

Comment:
**issue**: This is probably affecting the position of the bottom part and the alignment.

| Prebuild | Prebuild in Progress |
|-|-|
| <img width="1440" alt="prebuild-1" src="https://user-images.githubusercontent.com/120486/177721289-fb30925e-be81-4e00-9018-908bdb79228f.png"> | <img width="1440" alt="prebuild-2" src="https://user-images.githubusercontent.com/120486/177721300-9ec94e26-4152-4422-80ed-49448d774678.png"> |